docs/content/direct/multi-wec-aggregated-status.md

```diff
@@ -0,0 +1,191 @@
+# Proposal: Multi-WEC Aggregated Status Controller Enhancement
+
+**Status:** Proposal  
+**Related Issue:** [#2809 – KubeStellar controller logic to map back the status from WECs to READY values in WDSes](https://github.com/kubestellar/kubestellar/issues/2809)
+
+---
+
+## Current Scenario
+
+In the current KubeStellar implementation:
+
+### Case 1 — Single WEC and `wantSingletonReportedState` is true  
+The WDS workload `.status` is propagated from the single WEC.
+
+### Case 2 — Multiple WECs and `wantSingletonReportedState` is false  
+For workloads deployed to multiple WECs, the `.status` field in the WDS remains empty, resulting in no aggregated status propagated.
+
+This leads to limited visibility in ArgoCD, where workloads appear as *OutOfSync* or *Unknown*, even though all WECs are healthy.
+
+**Reference:**  
+- [PR #3297](https://github.com/kubestellar/kubestellar/pull/3297)  
+- [Multi-cluster example in docs](https://docs.kubestellar.io/release-0.28.0/direct/example-scenarios/#scenario-1-multi-cluster-workload-deployment-with-kubectl)
+
+---
+
+## Goal
+
+KubeStellar already collects per-WEC status in `WorkStatus` objects. This enhancement propagates that aggregated information into the `.status` field of the original workload object in the WDS, enabling ArgoCD and similar tools to observe combined readiness and health across all WECs.
+
+Key objectives:  
+- Maintain backward compatibility when `wantMultiWECReportedState` is false, preserving existing behavior.  
+- When `wantMultiWECReportedState` is enabled, provide fallback to the singleton mechanism if only one WEC is bound.  
+- Aggregate per-WEC `WorkStatus` objects and update the `.status` field in WDS workload objects with combined readiness and health status.
+
+---
+
+## Solution Approach
+
+### 1. Extend BindingPolicy CRD
+
+The configuration bit `wantMultiWECReportedState` is added under each `DownsyncModulation` rule (not directly under `BindingPolicy`), consistent with the canonical definition available at [./combined-status/#special-case-for-1-wec](./combined-status/#special-case-for-1-wec).
```

Comment:
`DownsyncModulation` is a Go language type; in particular, a struct type. It is not a "rule" --- except in the sense that any data type is a rule for the structure of some data, and in the sense that the semantics for any data type is a rule for how that data is processed; since these senses of "rule" attach to every data type, it is more helpful identify `DownsyncModulation` as a data type.